components/tooltip/README.md

```diff
@@ -7,20 +7,30 @@ Tooltips display additional information when users focus or hover on a point of
 <script type="module">
   import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/tooltip/tooltip.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip-help.js';
 
   window.addEventListener('load', function () {
     setTimeout(function() {
-      var tooltip = document.querySelector('#tooltip');
+      const tooltip = document.querySelector('#tooltip');
+      const helpTooltip = document.querySelector('#help-tooltip').shadowRoot.querySelector('d2l-tooltip');
```

Comment:
Agreed yeah. Doesn't look like it, but I think it would be worth adding a `showing` attribute to it that maps down to `d2l-tooltip`, even if it's only helpful for demo/test purposes.

Reply:
Yeah, I agree that accessing the `shadowRoot`, and having that show in the README, felt off. `showing` attribute added 👍